cmake/modules/configuration_files.cmake

```diff
@@ -76,10 +76,19 @@ zephyr_boilerplate_watch(CONF_FILE)
 
 zephyr_get(DTC_OVERLAY_FILE SYSBUILD LOCAL)
 
+# Look for SoC overlay files, these should be prepended before board overlay files
+# but after app.overlay, depending upon which files have been found
```

Comment:
this comment is a bit confusing, because it gives the reader the impression that a qualifier overlay will be used together with an `app.overlay`, which is not how the implementation works.

Reply:
Just had a second look.
Can see code comment is correct according to code (have code or my 👀 changed since earlier review ?) .

I'm not convinced that qualifier overlay should be used together with app.overlay.
If we have qualifier overlays behave identical to board overlay, then it works as an opt-out of an app.overlay, but with current proposal the app.overlay will still be included.

If following the board overlay approach, then a qualifier overlay can always do an
```
#include "app.overlay"
```
if it wants to include the `app.overlay`.